### PR TITLE
Added Delegation UI

### PR DESCRIPTION
**Pull request by TomSesselmann**

### CODE REVIEW

**Comment by TomSesselmann** [comment]
I think there might be something weird with my `package-lock.json`. is 18,000 additions normal?

**Comment by XertroV** [comment]
@TomSesselmann - you shouldn't include package-lock.json

File: web/src/SecureVote/Components/UI/Collapsible.elm [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+                []
+            else
+                body
+    in
+    column (w Collapsible)
+        [ spacing cmdSpacing ]
+    <|
+        [ el (w Title) [ onClick onCollapse, vary BtnCursor True ] <| text <| collapsedSymbol ++ " " ++ header
+        ]
+            ++ collapsedSection
```
Comment (TomSesselmann) [inline web/src/SecureVote/Components/UI/Collapsible.elm]: I found a cool new function the other day. You can replace this with just 

```
[ el (w Title) ...
, when isCollapsed body
]
```

http://package.elm-lang.org/packages/mdgriffith/style-elements/4.3.0/Element#when

Comment (XertroV) [reply web/src/SecureVote/Components/UI/Collapsible.elm]: 👍 awesome - good find

**Pull request merged**